### PR TITLE
fix typo src

### PR DESCRIPTION
**Pull request by coco875**

### CODE REVIEW

**Comment by coco875** [comment]
not match for now

File: src/code_80091750.c [hunk 1]
```diff
@@ -2461,7 +2461,7 @@ void func_80094A64(struct GfxPool *pool) {
     gMatrixHudCount = 0;
     gMatrixEffectCount = 0;
     gSPViewport(gDisplayListHead++, VIRTUAL_TO_PHYSICAL(&D_802B8880));
-    gDPSetScissor(gDisplayListHead++, G_SC_NON_INTERLACE, 0, 0, 320, 240);
+    gDPSetScissor(gDisplayListHead++, G_SC_NON_INTERLACE, 0, 0, SCREEN_WIDTH, 240);
```
Comment (MegaMech) [inline src/code_80091750.c:2464]: missed a screen height here

File: src/code_80005FD0.c [hunk 2]
```diff
@@ -1406,8 +1406,7 @@ void func_800090F0(s32 playerId, Player *player) {
 f32 func_80009258(UNUSED s32 playerId, f32 arg1, f32 arg2) {
     f32 temp_f2 = D_8016344C - arg2;
     f32 temp_f12 = arg1 - D_8016344C;
-
-    return gCourseTimer - ( (0.01666666f * temp_f2) / (temp_f2 + temp_f12) );
+    return gCourseTimer - ( (COURSE_TIMER_ITER * temp_f2) / (temp_f2 + temp_f12) );
```
Comment (MegaMech) [inline src/code_80005FD0.c:1409]: This might be the line that makes it not match.

It's possible the f at the end is required. Or num 6's are diff

**Review by MegaMech — DISMISSED** [review]

File: src/code_80091750.c (hunk 1 above)
Comment (coco875) [reply src/code_80091750.c:2464]: oh yeah forgot this

**Comment by coco875** [comment]
now match

**Review by MegaMech — APPROVED** [review]

**Pull request merged**